cli/units_cmd.go

```diff
@@ -111,6 +118,24 @@ func scanUnitsIntoConfig(cfg *config.Repository, quiet bool) error {
 	return nil
 }
 
+// checkRegularExecutableFile verifies that the file pointed to by name is a regular executable file.
+func checkRegularExecutableFile(name string) error {
+	switch info, err := os.Stat(name); {
+	case err != nil && os.IsNotExist(err):
```

Comment:
`err != nil && os.IsNotExist(err)` can be simplified to just `os.IsNotExist(err)` because that func is safe to run on `nil` error, it will return false.